nuxt.config.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
   /*
   ** Global CSS
   */
-  css: [],
+  css: ['./assets/scss/style'],
```

Comment:
what does this do?

Reply:
This was in layouts/default.vue. It is the overall styling for the application. Sadly it was causing issues when running tests. Unit tests could not find the font icons.. a bit strange.

To fix the issue, I placed the styling here, where it should technically be I guess.